inc/actions.php

```diff
@@ -474,6 +479,7 @@ function act_revert($act){
 function act_redirect($id,$preact){
     global $PRE;
     global $TEXT;
+  
```

Comment:
It looks like you need to set your editor to remove white-space at the end of lines